.textlintrc.json

```diff
@@ -115,12 +119,16 @@
         "LED",
         "Let's Encrypt",
         "Life360",
+        "LinkedIn",
         "Linky",
         "Linux",
         "Lovelace",
+        "MacBook",
         "macOS",
+        "Markdown",
```

Comment:
Maybe also `iMac`, `iPhone`, `iPad`, `iPod`

Reply:
Sure, tons of things! It is not the goal of this PR the be complete, but to make steps forward.